coralproject/talk
# Add German translations for notifications

## What does this PR do?

Add German translations for notifications.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

None

## How do I test this PR?

- enable DSA in admin settings
- write a comment
- reject the comment
- check notifications

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

As usual
